tests/lsan.supp

```diff
@@ -9,16 +9,16 @@ leak:^IoAdapter_init$
 leak:^load_server_config$
 
 # to be triaged; system_tests_autolinks
-leak:^qdr_error_description$
+leak:^qdr_error_description
```

Comment:
This is funny ASAN inconsistency. In C++, the suppression file entry is matched against function signature together with parameters, so the `$`s prevent a match.

Reply:
Reported as https://github.com/google/sanitizers/issues/1504